main.py

```diff
@@ -83,18 +89,31 @@ def get_configs():
 # add CORS middleware
 app.add_middleware(
     CORSMiddleware,
-    allow_origins=CONFIG["cors_domains"],
+    allow_origins=CONFIG["CORS_DOMAINS"],
```

Comment:
```suggestion
    allow_origins=CONFIG["CORS_DOMAINS"].split(","),
```
`allow_origins` takes a list of strings, so we'll need to split the env with some delimiter